packages/cashier/src/pages/deposit/crypto-deposit/__tests__/crypto-deposit.spec.tsx

```diff
@@ -437,4 +455,37 @@ describe('<CryptoDeposit />', () => {
 
         expect(screen.getByText('RecentTransactions')).toBeInTheDocument();
     });
+
+    it('should show AlertBanner for minimum deposit when third-party payment processor is used (CoinsPaid)', () => {
+        const minimum_deposit = 2;
+        const currency = 'tUSDT';
+        const mockRootStore = {
+            client: {
+                currency: 'tUSDT',
+            },
```

Comment:
```suggestion
         const minimum_deposit = 2;
        const mock = mockStore({
            client: { currency: 'tUSDT' },
```
@aum-deriv Please use `mockStore` 🙏🏻